twisted_s3/client.py

```diff
@@ -53,10 +53,17 @@ def _make_request(
 
         hashed_payload = auth.compute_hashed_payload(payload)
 
-        host = "{bucket}.s3-{region}.amazonaws.com".format(
-            bucket=bucket,
-            region=region
-        )
+        host = ""
+        if region == "us-east-1":
+            host = "{bucket}.s3.amazonaws.com".format(
```

Comment:
Ah, I see the issue now. I had to read through the link on your other PR for a sec. I understand the problem now. Kinda sad that `s3-us-east-1.amazonaws.com` doesn't work. Seems like an easy fix on their part.